osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -143,11 +146,12 @@ public string StoredBookmarks
             Version
         }.Concat(Metadata?.SearchableTerms ?? Enumerable.Empty<string>()).Where(s => !string.IsNullOrEmpty(s)).ToArray();
 
-        public override string ToString()
-        {
-            string version = string.IsNullOrEmpty(Version) ? string.Empty : $"[{Version}]";
+        public override string ToString() => $"{Metadata ?? BeatmapSet?.Metadata} {VersionString}".Trim();
 
-            return $"{Metadata ?? BeatmapSet?.Metadata} {version}".Trim();
+        public RomanisableString ToRomanisableString()
+        {
+            var metadata = (Metadata ?? BeatmapSet?.Metadata)?.ToRomanisableString() ?? new RomanisableString(null, null);
+            return new RomanisableString($"{metadata.GetPreferred(true)} {VersionString}".Trim(), $"{metadata.GetPreferred(false)} {VersionString}".Trim());
```

Comment:
Would be nice to have a concatenation/join operation framework-side to do this instead of having to do this song and dance...

Reply:
> Would be nice to have a concatenation/join operation framework-side to do this instead of having to do this song and dance...

You mean something like `RomanisableString.Join(RomanisableString, string)` ?

Reply:
At a `LocalisableString` level even, if possible. Might be a bit tricky to achieve but would be pretty good to have usability-wise.